docs/backend-system/core-services/root-http-router.md

```diff
@@ -74,3 +76,5 @@ backend.add(
   }),
 );
 ```
+
+Note that requests towards `/api/*` will always fall through the `routes` if there is no matching plugin route, and never be caught by the configured `indexPath`.
```

Comment:
Maybe reword "fall through the routes", that made me stop for a sec and wonder what you meant